x/ibc/02-client/exported/exported.go

```diff
@@ -4,45 +4,108 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/cosmos/cosmos-sdk/codec"
 	evidenceexported "github.com/cosmos/cosmos-sdk/x/evidence/exported"
+	connectionexported "github.com/cosmos/cosmos-sdk/x/ibc/03-connection/exported"
+	channelexported "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
 	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
 )
 
+// ClientState defines the required common functions for light clients.
+type ClientState interface {
+	GetID() string
+	ClientType() ClientType
+	GetLatestHeight() uint64
+	IsFrozen() bool
+
+	// State verification functions
+
+	VerifyClientConsensusState(
```

Comment:
Are these verification functions the ones needed to complete the handshakes? Might be helpful to annotate the methods with the different places they are anticipated to be used.